docs/upgrade-notes/migrate-tea-standalone.md

```diff
@@ -0,0 +1,148 @@
+---
+id: migrate_tea_standalone
+title: Migrate TEA deployment to standalone module
+hide_title: false
+---
+
+## Background
+
+Previous versions of Cumulus included deployment of the Thin Egress App (TEA) by default in the `distribution` module. As a result, this forced Cumulus users who wanted to deploy a new version of TEA to wait on a new release of Cumulus that incorporated that release.
+
+In order to give Cumulus users the flexibility to deploy newer versions of TEA whenever they want, deployment of TEA has been removed from the `distribution` module and **Cumulus users must now add the TEA module to their deployment**. [Guidance on integrating the TEA module to your deployment is provided](deployment/thin-egress-app.md), or you can refer to [Cumulus core example deployment code](https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/main.tf).
+
+By default, when upgrading Cumulus and moving from TEA deployed via the `distribution` module to deployed as a separate module, your API gateway for TEA would be destroyed and re-created, which could cause outages for any Cloudfront endpoints pointing at that API gateway.
+
+These instructions outline how to modify your state to preserve your existing Thin Egress App (TEA) API gateway when upgrading Cumulus and moving deployment of TEA to a standalone module.
```

Comment:
We should more explicitly note this is optional if you don't care about Cloudfront.

Reply:
https://github.com/nasa/cumulus/pull/1863/commits/90d61f20f71e08940a1e2b9a104f83e8f3bc4e8e#diff-190568db1790300e626312496836b296R17